bigquery/google/cloud/bigquery/client.py

```diff
@@ -1929,6 +1935,61 @@ def list_rows(
         )
         return row_iterator
 
+    def _schema_from_json_file_object(self, file):
```

Comment:
Nit: since `file` is a built-in, use `file_` or `file_obj` as the name.

Reply:
Changed name as directed.